pkg/cmd/openshift-kube-apiserver/admission/customresourcevalidation/securitycontextconstraints/defaulting_scc.go

```diff
@@ -67,14 +64,9 @@ func (a *defaultSCC) Admit(attributes admission.Attributes, o admission.ObjectIn
 		return err
 	}
 
-	internalSCC := uncastObj.(*securityapi.SecurityContextConstraints)
-	outSCCExternal := &securityv1.SecurityContextConstraints{}
-	if err := a.scheme.Convert(internalSCC, outSCCExternal, nil); err != nil {
-		return apierrors.NewForbidden(attributes.GetResource().GroupResource(), attributes.GetName(), err)
-	}
-
+	outSCCExternal := uncastObj.(*securityv1.SecurityContextConstraints)
 	defaultedBytes, err := runtime.Encode(a.codecFactory.LegacyCodec(securityv1.GroupVersion), outSCCExternal)
-	if err := a.scheme.Convert(internalSCC, outSCCExternal, nil); err != nil {
```

Comment:
ooooh, this.

Yeah, we need a defaulting call.  
1. Set up a direct call to the defaulting method and a unit test that proves equivalence.  (pull just with this).
2. Copy the defaulting call out so it only relies on external types. (pull just with this that includes the old unit test)
3. Remove the reference to internal.